translate/storage/lisa.py

```diff
@@ -164,15 +173,7 @@ def settarget(self, text, lang='xx', append=False):
                         pass
                 languageNode.text = text
         else:
-            self.set_target_dom(None, False)
-
-    def gettarget(self, lang=None):
-        """retrieves the "target" text (second entry), or the entry in the
-        specified language, if it exists"""
-        return self.getNodeText(self.get_target_dom(lang),
-                                getXMLspace(self.xmlelement,
-                                            self._default_xml_space))
-    target = property(gettarget, settarget)
```

Comment:
As a clarification, this `gettarget` should be kept, and not deprecated, since they have this extra `lang` argument. But the `gettarget` and such with no arguments that are present in the rest of TTK must be marked as deprecated in favor of their properties.